i18n/fr/docusaurus-plugin-content-docs/version-21.10/getting-started/mysql_tuto.md

```diff
@@ -0,0 +1,72 @@
+---
+id: monitor-mysql-server
+title: Superviser une base de données MySQL ou MariaDB
+---
+
+## Prérequis
+
+L'objectif de ce tutoriel est de superviser une base de données Mysql/MariaDB.
+
+Afin de pouvoir collecter les informations nécessaires à partir de la base de données, un utilisateur de la base de données disposant de privilèges spécifiques sera nécessaire. Si la base que vous souhaitez superviser est une base Centreon, un utilisateur dédié existe déjà (il s'appelle **centreon**). Dans le cas contraire, commencez par créer un nouvel utilisateur : connectez-vous à votre base de données, puis exécutez la commande suivante :
+
+```shell
+CREATE USER 'username'@'IP_POLLER' IDENTIFIED BY 'password';
+```
+
+Puis, que votre base soit une base Centreon ou non, exécutez la commande suivante :
+
+```shell
+grant ALL PRIVILEGES on *.* to 'centreon'@'IP_POLLER' identified by 'password'
```

Comment:
GRANT ALL PRIVILEGES ?
Looks like kind overkill and not secure :sweat: 
Only SELECT right should be granted.
not necessary to precise again the password.